tutorials/tutorial08_network_templates.ipynb

```diff
@@ -0,0 +1,328 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Tutorial 08: Networks from Custom Templates\n",
+    "\n",
+    "In the previous tutorial, we discussed how OpenStreetMap files can be simulated in Flow. These networks, however, may at time be imperfect, as we can see in the toll section of the Bay Bridge (see the figure below). The simulators SUMO and Aimsun both possess methods for augmenting the network after they have been imported, and store the changes in their own versions of the initial template (whether it was generated via a custom scenario class or a network imported from OpenStreetMap). In order to utilize these newly generated networks, we demonstrate in this tutorial how simulator-generated template files can be imported when running a simulation in Flow. \n",
+    "\n",
+    "<img src=\"img/osm_to_template.png\">\n",
+    "<center> **Figure 1**: Example benefit of converting OpenStreetMap to a custom template </center>\n",
+    "\n",
+    "The remainder of the tutorial is organized as follows. In section 1, we begin by importing the classic set of parameters. In section 2, we introduce the template files that are used as examples for importing the template files. In section 3, we present how custom SUMO network templates, i.e. the generated .net.xml files, can be modified and simulated in Flow for the purposed of improving network features. Finally, in section 4, we demonstrate how custom Aimsun network files can be simulated in Flow.\n",
+    "\n",
+    "## 1. Importing Modules\n",
+    "\n",
+    "Before we begin, let us import all relevant Flow parameters as we have done for previous tutorials. If you are unfamiliar with these parameters, you are encouraged to review tutorial 1."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# the TestEnv environment is used to simply simulate the network\n",
+    "from flow.envs import TestEnv\n",
+    "\n",
+    "# the Experiment class is used for running simulations\n",
+    "from flow.core.experiment import Experiment\n",
+    "\n",
+    "# the base scenario class\n",
+    "from flow.scenarios import Scenario\n",
+    "\n",
+    "# all other imports are standard\n",
+    "from flow.core.params import VehicleParams\n",
+    "from flow.core.params import NetParams\n",
+    "from flow.core.params import InitialConfig\n",
+    "from flow.core.params import EnvParams\n",
+    "from flow.core.params import SumoParams\n",
+    "\n",
+    "# create some default parameters parameters\n",
+    "env_params = EnvParams()\n",
+    "sim_params = SumoParams(render=True, sim_step=1)\n",
+    "initial_config = InitialConfig()\n",
+    "vehicles = VehicleParams()\n",
+    "vehicles.add('human', num_vehicles=1)"
```

Comment:
Not necessary for Aimsun section (although it doesn't alter anything), maybe move to SUMO section?